cmd/collector/main_test.go

```diff
@@ -28,6 +28,25 @@ import (
 	"gopkg.in/yaml.v3"
 )
 
+func TestGetDefaultCollectorConfigPathENV(t *testing.T) {
+	fakeConfigPath := "./fake/path/config.yaml"
+
+	os.Setenv(configPathENV, fakeConfigPath)
+	defer os.Unsetenv(configPathENV)
```

Comment:
Nit: Can we use https://pkg.go.dev/testing#T.Setenv instead? Does basically the same thing as these two lines, just built into the testing.T.

Reply:
I just used this because Mitch did too later in the test file, should I change those as well?

Reply:
I didn't know this was a thing and it does the `Unsetenv` for you. Looks like it was new in 1.17

Reply:
Outside the scope of this but we might want to update other ENV tests to use this if they aren't already.